src/scripts.js

```diff
@@ -1 +1,33 @@
-console.log("Hello World");
\ No newline at end of file
+console.log("Hello World");
+
+const userCard = document.querySelector('.user-card');
+
+window.addEventListener('load', displayUserData);
+
+function displayUserData() {
+  const user = new User(userData[33])
+  const userRepo = new UserRepo(userData)
+  const friendList = user.friends.map(friend => {
+  const info = userRepo.searchUsersByID(friend)
+    return info.name
+  })
+
+  const greeting = document.querySelector('.greeting');
```

Comment:
Good move putting these querySelectors() inside the local scope of the function! It's my habit to have all querySelectors() be global at the start of a project, and that leads to a lot of time consuming refactoring down the line. Good lookin' out for prefactoring!